src/blockdata/constants.rs

```diff
@@ -47,6 +47,8 @@ pub const MIN_TRANSACTION_WEIGHT: u32 = 4 * 60;
 /// The factor that non-witness serialization data is multiplied by during weight calculation
 pub const WITNESS_SCALE_FACTOR: usize = 4;
 
+/// Maximum weight of a transaction for it to be relayed by most nodes on the network
+pub const MAX_STANDARD_TRANSACTION_WEIGHT: u32 = 400_000;
```

Comment:
Probably we should match Bitcoin Core name `MAX_STANDARD_TX_WEIGHT` since it was the policy defined by @apoelstra which we should continue to follow

https://github.com/bitcoin/bitcoin/blob/4a540683ec40393d6369da1a9e02e45614db936d/src/policy/policy.h#L24